manifests/kubeedge/fluentbit-fluentBit.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: FluentBit
+metadata:
+  name: fluent-bit
+  namespace: fluent
+  labels:
+    app.kubernetes.io/name: fluent-bit
+spec:
+  image: kubesphere/fluent-bit:v1.9.4
+  positionDB:
+    hostPath:
+      path: /var/lib/fluent-bit/
+  resources:
+    requests:
+      cpu: 10m
+      memory: 25Mi
+    limits:
+      cpu: 500m
+      memory: 200Mi
+  fluentBitConfigName: fluent-bit-config
+  tolerations:
+    - operator: Exists
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: node-role.kubernetes.io/edge
+            operator: Exists
+  hostNetwork : true
```

Comment:
To collect edge node metrics from within a pod, you'll need to mount host /proc/ and /sys/ into the pod.
Please take a look at https://github.com/fluent/fluent-operator/blob/master/docs/plugins/fluentbit/clusterinput/node-exporter-metrics.md and test again
So you'll need to add the following content:
```yaml
  volumes:
    - name: hostProc
      hostPath:
        path: /proc/
    - name: hostSys
      hostPath:
        path: /sys/
  volumesMounts:
    - mountPath: /host/sys
      mountPropagation: HostToContainer
      name: hostSys 
      readOnly: true
    - mountPath: /host/proc
      mountPropagation: HostToContainer
      name: hostProc 
      readOnly: true
```